projects/aca-content/src/lib/components/sidenav/sidenav.component.html

```diff
@@ -1,32 +1,44 @@
-<div class="sidenav">
+<div class="sidenav" *ngIf="!hideSidenav">
   <ng-container [ngSwitch]="mode">
-    <div class="section action-menu" [ngClass]="'section--' + mode">
-      <app-main-action [expanded]="mode === 'expanded'"></app-main-action>
-      <app-create-menu [expanded]="mode === 'expanded'"></app-create-menu>
+    <div class="sidenav-header">
+      <div class="sidenav-header-title">
+        <div class="sidenav-header-title-logo" (click)="toggleClick()" (keypress)="toggleClick()">
+          <mat-icon svgIcon="workspace"
```

Comment:
the icon, logo and navigation paths are incorrectly set... if you check the original component that you rework, it has configuration support:
```
this.appName$ = store.select(getAppName);
    this.logo$ = store.select(getLogoPath);
    this.landingPage = this.appConfigService.get('landingPage', '/personal-files');
```